README.rst

```diff
@@ -11,27 +11,27 @@ dask-ndinterp
         :target: https://anaconda.org/conda-forge/dask-ndinterp
         :alt: conda-forge
 
-.. image:: https://img.shields.io/travis/jakirkham/dask-ndinterp/master.svg
-        :target: https://travis-ci.org/jakirkham/dask-ndinterp
+.. image:: https://img.shields.io/travis/dask-image/dask-ndinterp/master.svg
+        :target: https://travis-ci.org/dask-image/dask-ndinterp
         :alt: Travis CI
 
 .. image:: https://readthedocs.org/projects/dask-ndinterp/badge/?version=latest
         :target: https://dask-ndinterp.readthedocs.io/en/latest/?badge=latest
         :alt: Read the Docs
 
-.. image:: https://coveralls.io/repos/github/jakirkham/dask-ndinterp/badge.svg
-        :target: https://coveralls.io/github/jakirkham/dask-ndinterp
+.. image:: https://coveralls.io/repos/github/dask-image/dask-ndinterp/badge.svg
+        :target: https://coveralls.io/github/dask-image/dask-ndinterp
         :alt: Coveralls
 
-.. image:: https://img.shields.io/github/license/jakirkham/dask-ndinterp.svg
+.. image:: https://img.shields.io/github/license/dask-image/dask-ndinterp.svg
         :target: ./LICENSE.txt
         :alt: License
 
 
-A library for finding N-D interpolations of Dask Arrays
+A library for using N-D filters with Dask Arrays
```

Comment:
This line shouldn't have been changed. Fixing it in PR ( https://github.com/dask-image/dask-ndinterp/pull/2 ).